src/navbar.js

```diff
@@ -0,0 +1,38 @@
+import { Link } from 'react-router-dom';
+
+const Navbar = () => (
+  <nav className="nav-bar">
+    <Link
+      to="/"
+      className="logo"
+      style={{ color: 'rgb(1, 254, 1)', fontStyle: 'italic', fontSize: '1.6rem' }}
+    >
+      Math
+      <span> Magicians </span>
+    </Link>
+    <ul>
+      <li className="active">
+        <Link to="/">Home</Link>
+      </li>
+      <li>
+        <Link to="/calculator">Calculator</Link>
+      </li>
+      <li>
+        <Link to="/quote">Quote</Link>
+      </li>
+    </ul>
+  </nav>
```

Comment:
- [ ] [Optional] [Highly Recommended] Nice job creating a separate module for the navbar and importing it to `app.js`. For best practices, this is supposed to be a component, so kindly put it inside the `/component` directory.